apps/web/src/components/Notifications/useClearNotifications.tsx

```diff
@@ -23,11 +23,7 @@ export function useClearNotifications() {
         const connectionIds = [
           ConnectionHandler.getConnectionID(
             `client:Viewer:${userId}`,
-            'ProfileDropdownFragment_notifications'
-          ),
-          ConnectionHandler.getConnectionID(
-            `client:Viewer:${userId}`,
-            'ProfileDropdownContentFragment_notifications'
+            'StandardSidebarFragment_notifications'
```

Comment:
Good find, not easy to find this shit, I wish it was type safe some how

Reply:
Starting up a thread with some Relay folks about this 

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/icIMHMJMNkXxGoH3BEkt/01d336da-ae39-4bdd-8d31-f660394d9986/image.png)